src/js/components/breadcrumbs/JobsBreadcrumbs.js

```diff
@@ -90,7 +93,11 @@ function getBreadcrumb(item, details = true) {
 const JobsBreadcrumbs = ({ tree, item, children, details = true }) => {
   const elements = tree
     .filterItems(function(currentItem) {
-      return item != null && item.getId() === currentItem.getId();
+      return (
+        item != null &&
```

Comment:
Does it even make sense? If `item == null` then `filter => []` then `reduce => []`. Shall we extract that? I know that it was there in first place, but can we consider a little refactoring? 👼